hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -116,3 +122,13 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
       
       
 
+
+class JSBuilderSanityCheck extends JSBuilder:
```

Comment:
This should be a trait that lives alongside `JSBuilder`. We will want to use it for debug builds outside of the diff-tests.